docs/topics/auth/default.txt

```diff
@@ -999,6 +999,9 @@ implementation details see :ref:`using-the-views`.
       authenticated users accessing the login page will be redirected as if
       they had just successfully logged in. Defaults to ``False``.
 
+    * ``success_url_allowed_hosts``: A set of additional hosts that are safe
```

Comment:
> Missing versionchanged annotations and release notes. 

The class `LoginView` is entirely new to 1.11, not just this attribute. The class does have [versionadded](https://github.com/django/django/blob/9a2a52558e080f109a27d40a033a135c9d0e7e50/docs/topics/auth/default.txt#L974-L976) and [release notes](https://github.com/django/django/blob/9a2a52558e080f109a27d40a033a135c9d0e7e50/docs/releases/1.11.txt#L82-L84). Do you want to see this specific attribute in the release notes and annotated as well?